### DISCUSSION TITLE
461 Unsupported transport

### DISCUSSION THREAD

**Discussion by jamie-2bit** [Sunoo/homebridge-camera-ffmpeg#1141] [Q&A] [answered]
I've recently switched cameras to a Eufy 2k Indoor cam. 
Upon doing so, and updating the RTSP stream name in the config, I'm getting these errors:

[6/22/2021, 7:28:08 PM] [Camera FFmpeg] [PTZ Camera] [rtsp @ 0x25bd840] [error] method SETUP failed: 461 Unsupported transport

The stream will still show in the home app, so it does still work - I just want to know what could be causing this error.

Any ideas?

**Comment by Sunoo** [comment, marked as answer]
Please share your config.

**Reply by tjm7844** [reply]
Hello - I am having the same issue, with the same camera (just got this set up in advance of a hopeful HSV update soon - very exciting!):

Config is as follows: 
{
            "name": "CameraUI",
            "port": 8080,
            "atHomeSwitch": true,
            "theme": "auto",
            "language": "auto",
            "debug": false,
            "mqtt": {
                "active": false
            },
            "http": {
                "active": false,
                "localhttp": false
            },
            "ssl": {
                "active": false
            },
            "cameras": [
                {
                    "name": "EufyPTZTest",
                    "manufacturer": "Eufy",
                    "model": "Indoor Cam Pan&Tilt",
                    "serialNumber": "[SERIAL]",
                    "firmwareRevision": "2.1.0.6",
                    "unbridge": true,
                    "videoConfig": {
                        "source": "-i rtsp://[USER]:[PASSWORD]@[CAMIP]live0",
                        "stimeout": 0,
                        "audio": true
                    }
                }
            ],
            "platform": "CameraUI"
        }

**Reply by Sunoo** [reply]
Looks like you are using homebridge-camera-ui, not this plugin. That said, that was based on this plugin and still uses FFmpeg, so I can help somewhat.

Try adding `-rtsp_transport tcp` to the start of your source in your config.

**Reply by tjm7844** [reply]
My apologies! I've been bouncing back and forth between the two plugins recently, checking status on HSV, and got confused.

Your suggestion seemed to work. Thank you!

**Reply by Sunoo** [reply]
No worries, there isn’t a rivalry between us or anything. :P
